assignments/game-of-life/code.js

```diff
@@ -0,0 +1,54 @@
+drawFilledRect(0, 0, width, height, 'black')
+
+
+
+const makeArrayRow = () => {
+  let row = []
+  for (let i = 0; i < width; i += 5) {
+    if (Math.random() < .2) {
+      row.push(1)
+    } else {
+      row.push(0)
+    }
+  }
+  return row
+}
+const makeArray = () => {
+  let b = []
+  for (let i = 0; i < height / 5; i++) {
+    b.push(makeArrayRow())
+  }
+  return b;
+}
+
+
+
+const drawTheThings = (b) => {
+  for (let i = 0; i < height / 5; i++) {
+    for (let j = 0; j < width / 5; j++) {
```

Comment:
These two lines actually only work because we happen to have created the board to be `height / 5` rows and `width / 5` columns. But we can actually know how big the board is:

```suggestion
  for (let i = 0; i < b.length; i++) {
    for (let j = 0; j < b[i].length; j++) {
```